tests/integration/test_integration.py

```diff
@@ -4,6 +4,7 @@
 from bigchaindb import Bigchain
 
 
+# TODO remove as it is unused?
```

Comment:
This looks like the same thing as the one in the root conftest? We might as well remove it then.